lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "packages": [
+    "packages/*"
+  ],
+  "version": "independent"
```

Comment:
/random location to allow threaded messages.

We should probably get some README.md updates for how to dev locally with lerna / how to publish / etc. And if there are any recommendations for majors (let them major bump independently vs major bumping everything together).

Reply:
Good call out, the good part about independent is that we can for sure no longer bump something that didn't get a change. I will make a some README changes tonight. I got tied up w/ other things on my plate 😛

Reply:
Updated readme w/ dets. Going to merge as I also fixed a critical security patch form dependabot